### PR TITLE
Documentation for Taipy Studio GUI (#318)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #318.

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/studio/gui.md [hunk 1]
```diff
@@ -1,25 +1,177 @@
...
 
-- In a Markdown file (the filename extension must be `.md`);
-- In any string that appears in Python source code.
+The extension is activated when Visual Studio Code finds Markdown text:
 
-Beside the IntelliSense service, Taipy Studio also reports, in the *Problems*
-view, warnings and errors it finds in your Markdown content.
+- In a Markdown file (the filename extension must be `.md`) which is
+  part of the currently opened project;
+- In any string that appears in Python source code, part of the currently
```
Comment (dinhlongviolin1) [inline docs/manuals/studio/gui.md:28]: more accurately would be with triple quote strings

File: docs/manuals/studio/gui.md [hunk 2]
```diff
@@ -1,25 +1,177 @@
...
+Here is an example of how a control can be created in the Visual Studio Code
+text editor for a Markdown file:
+
+<p align="center">
+  <img src="../images/autocomplete_element.gif" width=75%>
+</p>
+
+### Access to variable and function names
+
+If the Markdown content is entered as a string value in a Python file or
```
Comment (dinhlongviolin1) [inline docs/manuals/studio/gui.md]:
```suggestion
If the Markdown content is entered as a triple-quote string value in a Python file or
```

**Review by dinhlongviolin1 — APPROVED** [review]
Thanks! Everything looks on point

File: docs/manuals/studio/gui.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md:28]: Ah didn't pay attention. Thanks!

File: docs/manuals/studio/gui.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/gui.md]: 👍

**Pull request merged**